scripts/sui-install.sh

```diff
@@ -1,5 +1,21 @@
-echo "Installing SUI..."
-# Keep the version in sync with Move.toml and Dockerfile
-cargo install --locked --git https://github.com/MystenLabs/sui.git --tag mainnet-v1.8.2 sui
+#!/usr/bin/env bash
+#
+# Installs the sui command line tool (if necessary)
+#
+
+# Keep the version in sync with Move.toml
+sui_version="1.8.2"
+
+if command -v sui &> /dev/null
+then
+    current_version=$(sui --version | cut -d' ' -f2 | cut -d '-' -f1)
+    if [ "$current_version" = "$sui_version" ]; then
+        echo "sui ${sui_version} is already installed"
```

Comment:
This is what allows us to run this script in CI without waiting for half an hour - it exits if the right sui version is already installed.